src/main/java/com/example/universe/simulator/entityservice/services/GalaxyService.java

```diff
@@ -29,6 +32,7 @@ public Page<Galaxy> getList(Specification<Galaxy> specification, Pageable pageab
         return repository.findAll(specification, pageable);
     }
 
+    @Cacheable(value= "Galaxy", key="#id")
```

Comment:
You should use a spring property or java constant instead of the hardcoded `Galaxy`. It's used in multiple places and will be hard to change if needed. Also, I don't think you need to specify the key parameter in this case, as it's automatically inferred from the method argument (same goes for the `delete` method).

Reply:
Done